packages/charting/configure/src/correct-response.jsx

```diff
@@ -216,14 +221,19 @@ export class CorrectResponse extends React.Component {
   }
 
   render() {
-    const { classes, model, charts } = this.props;
+    const { classes, model, charts, error, correctAnswerErrors } = this.props;
     const { categories } = this.state;
 
     return (
       <div>
         Define Correct Response
         <div className={classes.container}>
           <div className={classes.column} key="graph">
+            {correctAnswerErrors && (
+              <Typography component="div" type="body1" className={classes.noCorrectResponse}>
+              <span>{correctAnswerErrors.indenticalError || correctAnswerErrors.categoriesError}</span>
```

Comment:
Can't indenticalError and categoriesError be defined at the same time? If so, shouldn't we display both errors?

Reply:
no, if we get a categoriesError, then we won't get an error for identicalError